buildSrc/src/main/groovy/openhouse.iceberg-aws-conventions.gradle

```diff
@@ -1,12 +1,10 @@
 ext {
-  icebergAwsVersion = '1.5.2'
+  icebergAwsVersion = '1.2.0'
```

Comment:
glad this worked nicely! looking back, having a single iceberg version is much cleaner